packages/core-utils/src/coders/ecdsa-coder.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint prefer-arrow/prefer-arrow-functions: "off" */
```

Comment:
Instead of this could we either: 
1. actually comply with the rule?
2. get away with a small number of rule-specific `disable-next-line` comments?